word-count/word_count.rb

```diff
@@ -0,0 +1,24 @@
+class Phrase
+  attr_reader :phrase
+
+  def initialize(phrase)
+    @phrase = sanitize(phrase)
+  end
+
+  def word_count
+    words = phrase.split(' ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.